elixir-language-configuration.json

```diff
@@ -31,13 +33,96 @@
     { "open": "~S\"", "close": "\"" },
     { "open": "~T\"", "close": "\"" },
     { "open": "~U\"", "close": "\"" },
+    { "open": "~W\"", "close": "\"" },
     { "open": "~b\"", "close": "\"" },
     { "open": "~c\"", "close": "\"" },
     { "open": "~e\"", "close": "\"" },
     { "open": "~r\"", "close": "\"" },
     { "open": "~s\"", "close": "\"" },
     { "open": "~w\"", "close": "\"" },
+
+    { "open": "~B/", "close": "/" },
+    { "open": "~C/", "close": "/" },
+    { "open": "~D/", "close": "/" },
+    { "open": "~E/", "close": "/" },
+    { "open": "~F/", "close": "/" },
+    { "open": "~H/", "close": "/" },
+    { "open": "~L/", "close": "/" },
+    { "open": "~N/", "close": "/" },
+    { "open": "~R/", "close": "/" },
+    { "open": "~S/", "close": "/" },
+    { "open": "~T/", "close": "/" },
+    { "open": "~U/", "close": "/" },
+    { "open": "~W/", "close": "/" },
+    { "open": "~b/", "close": "/" },
+    { "open": "~c/", "close": "/" },
+    { "open": "~e/", "close": "/" },
+    { "open": "~r/", "close": "/" },
+    { "open": "~s/", "close": "/" },
+    { "open": "~w/", "close": "/" },
```

Comment:
I see Elixir LS's elixir language configuration escapes the slashes with backslashes (ie `{ "open": "~B\/", "close": "\/" }`). Is there a particular reason you didn't keep the same format?